### PR TITLE
3.6.4 release notes and KC updates

### LINKED ISSUE #3181: [Design Deliverable] Update Knowledge Center with Error Type guidance and Release Notes Email
**Description:**

Supports #3180

**AC:**

- [x] Error guide page is updated with error type guidance
- [x] Screenshots are updated
- [x] The design is usable and accessible, meaning it adheres to definition of done standards for design work.
> <details>
>      <summary>Expand Definition of Done Standards</summary>
> 
> - It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented
> 
> - Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented
> - It follows [accessibility guidelines](https://accessibility.digital.gov/) and accessibility implementation notes are documented (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.)
> - If feedback identifies bigger questions or unknowns, create additional issues to investigate
> </details>

**Tasks:**

- [x] Draft content addition in gitbook
- [x] Update screenshots
- [x] Draft PR

### PR DESCRIPTION
**Pull request by victoriaatraft**
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #3181 

Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/
and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

Screenshot 2024-10-07 at 12 51 25 PM
Screenshot 2024-10-07 at 12 50 58 PM

### CODE REVIEW

**Review by reitermb — APPROVED** [review]
💯 🚀

**Comment by klinkoberstar** [comment]
Thanks @victoriaatraft and @reitermb ! Here are my comments:

- The following line may be redundant given the addition of the table to the KC: _Inconsistency across one or more cases in a file will trigger this status, where at least some of the records in the file weren't added to the database._
- @lfrohlich  raised a good point that the excel interface will vary for different users which we should resolve before release. Perhaps we can use general language or direct users to the Support Center for filtering in Excel.

**Comment by ADPennington** [comment]
@victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.

**Comment by victoriaatraft** [comment]
> @victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.

@ADPennington hi here is the [Gitbook link for the Knowledge Center](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/knowledge-center-drafts/interpreting-error-types). The error type table and red text is what is new content on the Understanding Error Reports and File Structure page.

**Comment by klinkoberstar** [comment]
> > @victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.
> 
> @ADPennington hi here is the [Gitbook link for the Knowledge Center](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/knowledge-center-drafts/interpreting-error-types). The error type table and red text is what is new content on the Understanding Error Reports and File Structure page.

Thank you for linking this @victoriaatraft - so much easier to make comments here! I made some additional suggested edits and left comments to give some context about why I proposed changes. I've been in the weeds with this content for a bit though so open to suggestions about how best to share this info.

File: product-updates/knowledge-center/index.html [hunk 1]
```diff
@@ -337,6 +337,151 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
...
+            <div class="usa-icon-list__icon text-ink">
+              <img src="../img/added.svg" aria-hidden="true" alt="" style="background-color: #d3ffcc; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                Fiscal Quarter Table Explainer to the TDP Data Files Page
+              </h5>
+              <p>The records within your data files are organized by <b>calendar quarter period</b> which runs from January 1 to December 31. However, these files must be reported based on the fiscal quarter period, which runs from October 1 to September 30. For example, files that cover <b>Calendar Year 2024 Quarter 4</b> are to be reported for <b>Fiscal Year 2025 Quarter 1</b>, which is due by <i>February 14, 2025.</i></p>
+              <p>We've <b>added</b> the table below to the <a href="https://tanfdata.acf.hhs.gov/data-files">data files submission page</a> to help users map the calendar period content within the file to the reporting fiscal period.</p>
```
Comment (ADPennington) [inline product-updates/knowledge-center/index.html]: sorry the confusion but i thought this link would point to the KC page https://tdp-project-updates.app.cloud.gov/knowledge-center/uploading-data.html and that this page would be updated with how the page will look? 

And something i'm also just realizing -- was this change actually made to TDP yet? I dont recall approving a dev ticket for this change, but i did approve the design ticket related to this. 

If this change isnt actually included in a qasp-approved dev ticket, then we should table this note for a future release. @victoriaatraft @reitermb @klinkoberstar @lfrohlich

Comment (victoriaatraft) [reply product-updates/knowledge-center/index.html]: resolved on our call - content with this is moved for [future release notes](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/release-notes-workflow/future-release-notes) on gitbook.

**Comment by victoriaatraft** [comment]
@ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.

Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/
and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

**Comment by ADPennington** [comment]
> @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> 
> Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

@victoriaatraft couple of followups:

- i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC? 
- do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.

**Comment by victoriaatraft** [comment]
> > @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> > Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html
> 
> @victoriaatraft couple of followups:
> 
> * i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC?
> * do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.

@ADPennington agree. done 😊

**Comment by ADPennington** [comment]
> > > @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> > > Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html
> > 
> > 
> > @victoriaatraft couple of followups:
> > 
> > * i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC?
> > * do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.
> 
> @ADPennington agree. done 😊

@victoriaatraft thank you :) i did a quick comparison of the live KC (left) and the test KC (right), and some things moved around. Please see below:

Screenshot 2024-10-09 114306
Screenshot 2024-10-09 114033
Screenshot 2024-10-09 113923

**Comment by klinkoberstar** [comment]
Good catch by Alex on the example section - otherwise, the language looks good to me!

**Comment by ADPennington** [comment]
@victoriaatraft -- @klinkoberstar and I just took a final sweep of the release notes and KC updates and have a couple of final suggestions. Please ping us when these changes are pushed up to the test environment for our final review. cc: @lfrohlich @ttran-hub

- please change the release note date to Oct 10, 2024. I have completed testing of the release in staging and believe we will push the release to prod some time today. 
Screenshot 2024-10-10 123622


- Can we move the text referenced below above the table (in a new paragraph)?
Screenshot 2024-10-10 123553

**Comment by victoriaatraft** [comment]
@klinkoberstar @ADPennington done
cc: @lfrohlich @ttran-hub

**Review by klinkoberstar — APPROVED** [review]

**Review by ADPennington — APPROVED** [review]
LGTM 🥇 cc: @lfrohlich @ttran-hub

**Pull request merged**